snakegame.py

```diff
@@ -68,41 +73,48 @@ def playgame(self):
 		Runs the game. Allows to set up initial values.
 		Or holds space for adding menu.
 		"""
-		print("Runing new game.")
 
 		initial_length = 3
 		initial_speed = 3		
 		fruits_amount = 2
 
 		(total_length, total_time) = self.rungame(initial_length, initial_speed, fruits_amount)
 
-		print("Game finished.")
-
 		return (total_length, total_time)
+
+	def getPlayerInput(self):
+		# temporarily added some value. TODO: work on this
+		return 1
+
+	def applyPlayerInput(self, input):
+		print("<applyPlayerInput>")
+		# TODO: work on this
 	
 	def rungame(self, init_length, init_speed, fruits_no):
 		"""
 		Runs the game.
 		"""
+		print("Runing new game.")
 
 		# Samples only
 		self.snakeSample = Snake(3, 500, 500)
 		self.fruits = []
 		self.snakeSample.joints = [Joint(5,8), Joint(4,8), Joint(3,8)]
 		for i in range(fruits_no):
 			self.fruits.append(Joint(random.randrange(16), random.randrange(16)))
-		
+		speed = init_speed
+
 		while self.running:
-			# if inputfromplayer:
-			# 	playerinput = # get input from player
-			# 	applyinput()
-			# dostep(snake)
-			# wait(1.0/init_speed)
+			period = 1 / speed

```

Comment:
time.monotonic() is in seconds, right? Then this is correct 👍

Reply:
Yes, monotonic returns float number in seconds. Monotonic function purpose is to be get in one time and than compared in another, since reference point of the time is undefined. For more see https://docs.python.org/3/library/time.html